src/main/scala/org/holmesprocessing/totem/driver/driver.scala

```diff
@@ -138,6 +140,8 @@ object driver extends App with Instrumented {
           PEiDWork(key, uuid_filename, taskingConfig.default_service_timeout, "PEID", GeneratePartial("PEID"), li)
         case ("PEINFO", li: List[String]) =>
           PEInfoWork(key, uuid_filename, taskingConfig.default_service_timeout, "PEINFO", GeneratePartial("PEINFO"), li)
+        case ("SHODAN", li: List[String]) =>
+          ShodanWork(key, orig_filename, List(taskingConfig.default_service_timeout, 300).max, "SHODAN", GeneratePartial("SHODAN"), li)
```

Comment:
Reasoning, why I suggested this change to @jplaui : there is no way to have per-service timeouts configured in the totem.conf (yet?).
This change ensures that if the user sets a default service timeout of less than 5 minutes in the totem.conf then Shodan still has 5 minutes, but it enables the user to set a higher value in the totem.conf if necessary.

I don't think setting it to flat 5 minutes in `enumerateWork` is a really good idea. We should not hard-code an upper boundary for service timeouts. (Similar feelings about the half hour for VirusTotal)
At some point we need a way to have an (optional) timeout setting for services in the totem.conf to tackle this. (more setup complexity but should be same complexity in `enumerateWork`)

A better solution for this one for the time being might be `Math.max`? (less complexity? not sure about that)